.github/scripts/tests/therock_configure_ci_test.py

```diff
@@ -8,19 +8,27 @@
 import therock_configure_ci
 
 class ConfigureCITest(unittest.TestCase):
+    # Helper to get project names from a list of projects
+    def extract_project_names(self, projects):
+        return [p.strip() for project in projects for p in project["project_to_test"].split(',')] if projects else []
+
     @patch("subprocess.run")
     def test_pull_request(self, mock_run):
         args = {
             "is_pull_request": True,
-            "input_subtrees": "projects/rocprim\nprojects/hipcub"
         }
         
         mock_process = MagicMock()
-        mock_process.stdout = ""
+        mock_process.stdout = "projects/rocprim/src/main.cpp\nprojects/hipcub/src/main.cpp\nprojects/rocwmma/src/main.cpp"
         mock_run.return_value = mock_process
 
-        project_to_run = therock_configure_ci.retrieve_projects(args)
-        self.assertEqual(len(project_to_run), 1)
+        project_to_run, test_type = therock_configure_ci.retrieve_projects(args)
+        project_names = self.extract_project_names(project_to_run)
+        self.assertIn("rocprim", project_names)
```

Comment:
just curious, does self.assertIn work with string? if so, then you could do:

```suggestion
        self.assertIn("rocprim", project_to_run)
```

Reply:
So, this works:

```suggestion
        self.assertIn("rocprim", str(project_to_run))
```

which I like better than what I've got now.  Thanks :)